acceptance/auth/bundle_and_profile/output.txt

```diff
@@ -0,0 +1,32 @@
+
+=== Inside the bundle, no flags
+>>> errcode $CLI current-user me
+"$USERNAME"
+
+=== Inside the bundle, target flags
+>>> errcode $CLI current-user me -t dev
+"$USERNAME"
+
+=== Inside the bundle, target and matching profile
+>>> errcode $CLI current-user me -t dev -p DEFAULT
```

Comment:
nit: This doesn't provide any extra coverage since -t dev is implicit in the configuration. Same below.

Reply:
It does, because it test what happens if we provide both `-t` and `-p` with matching hosts.
because we could make a change making -t and -p exclusive and this test would break